build.gradle

```diff
@@ -18,11 +15,9 @@ apply plugin: 'org.flywaydb.flyway'
 
 jar {
     baseName = "Identity-service"
-    version = "0.1.0"
```

Comment:
eh, I think it's helpful to have even if you're not versioning it like a lib as it conveys extra information, but I see your point.  But yes, moving it outside of the ```jar``` expando's scope makes it a ```project``` property.